modules/swagger-codegen/src/main/resources/php/ApiClient.mustache

```diff
@@ -223,7 +223,16 @@ class ApiClient
 
         // Handle the response
         if ($response_info['http_code'] == 0) {
-            throw new ApiException("API call to $url timed out: ".serialize($response_info), 0, null, null);
+            $curl_error_message = curl_error($curl);
+
+            // curl_exec can sometimes fail but still return a blank message from curl_error().
+            if (!empty($curl_error_message)) {
+                $error_message = "API call to $url failed: ".curl_error($curl);
```

Comment:
Using `$curl_error_message` instead of another call to `curl_error($curl)` would improve readability slightly I think.